src/utils/dates.js

```diff
@@ -1,10 +1,5 @@
-export function formatIsoString(s) {
-  const options = {
-    month: 'short',
-    day: 'numeric',
-    year: 'numeric',
-    hour: '2-digit',
-    minute:'2-digit'
-  };
-  return new Date(s).toLocaleString([], options);
+import moment from 'moment';
+
+export function formatTime(s) {
+  return moment(s).format('MMM D, YYYY, h:mm A');
```

Comment:
I think that is my point this format is different to the format we are using else where for example, [here](https://github.com/OpenChemistry/mongochemclient/blob/master/src/components/notebooks/table.js#L61), it provides a must more human friendly format.

Reply:
This has been modified to use the same kind of format as the notebooks use. It was also added as a utils function for consistency throughout the code.